scripts/deploy_generator.ts

```diff
@@ -112,7 +114,9 @@ async function uploadAndInitGenerator(terra: LCDClient, wallet: any) {
                 tokens_per_block: String(8403094),
                 vesting_contract: network.vestingAddress,
                 factory: network.factoryAddress,
-                whitelist_code_id: network.treasuryCodeID
+                whitelist_code_id: network.treasuryCodeID,
```

Comment:
It would be better to use name `whiteListCodeID` for the stored code of the whitelist. Using the code for the treasury is a case